portfolio/src/main/webapp/script.js

```diff
@@ -73,3 +75,15 @@ function drawChart(limitMap) {
   const chart = new google.visualization.PieChart(document.getElementById('chart-div'));
   chart.draw(data, options);
 }
+
+function fetchBlobUrlAndShowForm() {
+  fetch('/blobstore-url')
+      .then((response) => {
```

Comment:
You can do one line lambda such as `then(response => response.text())`

Reply:
Alright sure Tong.